slam-libraries/viam-cartographer/src/slam_service/config.cc

```diff
@@ -94,6 +104,23 @@ void ParseAndValidateConfigParams(int argc, char** argv,
     slamService.data_rate_ms = std::chrono::milliseconds(FLAGS_data_rate_ms);
     slamService.map_rate_sec = std::chrono::seconds(FLAGS_map_rate_sec);
 
+    // TODO: Remove once PR #1689 is submitted
+    // https://github.com/viamrobotics/rdk/pull/1689 This will allow intergation
+    // tests to pass (See associated JIRA ticket:
+    // https://viam.atlassian.net/browse/RSDK-1593)
+    if (FLAGS_delete_processed_data.empty()) {
+        slamService.delete_processed_data = false;
+    } else {
+        slamService.delete_processed_data =
+            !(FLAGS_delete_processed_data == "false");
+    }
```

Comment:
I was torn on this one because its a string so I need to check spelling of the word and choose to prioritize the true case (such that if a user wrote 'fales' it would trigger true). However since RDK will always spell true/false correctly I agree with you we can stick to the more legible version